refs.bib

```diff
@@ -0,0 +1,8 @@
+@article{Doe2016,
+  author = {Doe, J and Smith, R},
+  journal = {Title of the journal},
+  pages = {10--30},
+  title = {Title of the paper},
```

Comment:
Strange, on my system, using natbib and `\bibliographystyle{elsarticle-harv}`, upper cases get converted to lower in the reference list if not protected by `{}`.
I tried a few other `\bibliographystyle{}` to check whether it was a `elsarticle-harv` only styling, but same.